daffodil-cli/src/it/scala/org/apache/daffodil/executing/TestCLIexecuting.scala

```diff
@@ -123,27 +123,27 @@ class TestCLIexecuting {
   //    }
   //  }
 
-  @Test def test_993_CLI_Executing_Listing_listAll() {
-    val tdmlFile = Util.daffodilPath("daffodil-test/src/test/resources/org/apache/daffodil/section06/entities/Entities.tdml")
-    val testTdmlFile = if (Util.isWindows) Util.cmdConvert(tdmlFile) else tdmlFile
-
-    val shell = Util.start("")
-
-    try {
-      shell.sendLine(String.format("%s test -l %s", Util.binPath, testTdmlFile))
-      shell.expect(contains(output16))
-      shell.sendLine()
-
-      val countTests = if (Util.isWindows) "find /v \"\" /c" else "wc -l"
-      val numTests = Integer.parseInt((String.format("%s test -l %s", Util.binPath, testTdmlFile) #| countTests !!).trim())
-      val numFile = Integer.parseInt((String.format(if (Util.isWindows) "cmd.exe /c more %s | find /c \"parserTestCase>\"" else "grep -c parserTestCase> %s", testTdmlFile)!!).trim())
-
-      assertTrue("Number of tests run should match the number of tests in the file.", numTests == numFile)
-      shell.sendLine("exit")
-    } finally {
-      shell.close()
-    }
-  }
+//  @Test def test_993_CLI_Executing_Listing_listAll() {
```

Comment:
Why keep this? Probably should just delete.

Reply:
This is actually broken on Azure Pipelines on Windows, but works fine on linux. I couldn't figure out the problem so I temporarily commented it out and just forgot about it. I'll see if I can fix it, and if not I'll reenable it and then we can deal with it when ASF allows Azure Pipelines support.

Reply:
Actually, this test was just trying to be too smart. I've simplfied it a bit while still sufficiently testing listing tdml files. I've uncommented it and it should now run on windows.